ethereumj-core/src/main/java/org/ethereum/util/RLP.java

```diff
@@ -547,20 +461,33 @@ public static byte getCommandCode(byte[] data) {
      */
     public static RLPList decode2(byte[] msgData) {
         RLPList rlpList = new RLPList();
-        fullTraverse(msgData, 0, 0, msgData.length, 1, rlpList);
+        fullTraverse(msgData, 0, 0, msgData.length, rlpList, Integer.MAX_VALUE);
         return rlpList;
     }
 
+    /**
+     * Decodes without going deep after 1st level list
+     * (actually, 2nd as RLP nesting one extra level)
+     * @param msgData rlp data
+     * @return list of RLPItems
+     */
+    public static RLPList decode2ListNarrow(byte[] msgData) {
```

Comment:
Let's use `unwrapList` name for this method. Short and addresses its meaning pretty well

Reply:
I'd also update description with an example of how does it work.